apps/admin-portal/package.json

```diff
@@ -29,7 +29,9 @@
         "@wso2is/validation": "^1.0.2"
     },
     "devDependencies": {
-        "run-script-os": "^1.0.7"
+        "fork-ts-checker-webpack-plugin": "^4.0.3",
```

Comment:
Shall we pull it from the root package.json? And use as a peerDependency.

Reply:
Resolved with commit `Fix peer dependencies` commit.